package-lock.json

```diff
@@ -2975,7 +2975,6 @@
 			"resolved": "https://registry.npmjs.org/align-text/-/align-text-0.1.4.tgz",
 			"integrity": "sha1-DNkKVhCT810KmSVsIrcGlDP60Rc=",
 			"dev": true,
-			"optional": true,
```

Comment:
It looks like this changes to the package-lock file were unintentional. If that's the case would it be possible to remove them?

Reply:
My bet is that `npm --version` will print version lower than 6.7.0 where this change in behavior was introduced. Changes to this file should be reverted in this PR.

Reply:
I've been a little under the weather, but I try and revert this back to you the master sometime this afternoon.

Reply:
You can copy the content of the file in the repository and paste into your branch. That's the easiest approach you can take.